app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class SessionsController < ApplicationController
+  def new; end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
+
+    if user&.authenticate(params[:session][:password])
+      # Log the user in and redirect to the user's show page.
+      cookies.permanent[:user_id] = user.id
+      cookies.permanent[:remember_token] = user.remember_token
+      flash[:success] = 'Welcome!'
+      redirect_to posts_path
+    else
+      # Create an error message.
+      flash[:error] = 'Wrong combination'
+      redirect_to signup_path
+    end
+  end
+
+  def destroy
+    cookies.delete :user_id
+    cookies.delete :remember_token
+
+    redirect_to signup_path
```

Comment:
As said by @Ispirett 

> This method should set the @current_user to nil [requirement](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/session_delete_requirement.png)

Please, read the requirement in the odin project in [this screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/session_delete_requirement.png)